lib/supply/uploader.rb

```diff
@@ -74,6 +74,14 @@ def upload_binary
         else
           client.upload_apk_to_track(Supply.config[:apk], Supply.config[:track])
         end
+
+        ## Obbs upload
+        search = File.join(File.dirname(Supply.config[:apk]), '*.obb')
```

Comment:
I think it's time to extract the Supply.config[:apk] into a variable inside the upload_binary function